src/queries/incremental-compilation-in-detail.md

```diff
@@ -533,5 +533,5 @@ be reusable. See <https://github.com/rust-lang/rust/issues/47389> for more
 information.
 
 
-
 [query-model]: ./query-evaluation-model-in-detail.html
+[try_mark_green]: https://doc.rust-lang.org/nightly/nightly-rustc/src/rustc_query_system/dep_graph/graph.rs.html#574-599
```

Comment:
It's probably best not to link to the line numbers since they'll constantly change.

```suggestion
[try_mark_green]: https://doc.rust-lang.org/nightly/nightly-rustc/src/rustc_query_system/dep_graph/graph.rs.html
```